workflows/differentialabundance.nf

```diff
@@ -157,8 +171,44 @@ workflow DIFFERENTIALABUNDANCE {
         ch_versions = ch_versions
             .mix(AFFY_JUSTRMA_RAW.out.versions)
 
-    }
-    else if(params.study_type == 'geo_soft_file'){
+    } else if (params.study_type == 'maxquant'){
+        // For maxquant, run proteus module to import the protein abundances
+
+        // Save contrasts file to channel
+        ch_contrasts_file = Channel.from([[exp_meta, file(params.contrasts)]])
```

Comment:
This is already defined below the conditional - just relocate that to above the conditional so you can use it here